test/camelcase/sub-dir/a-sub.js

```diff
@@ -0,0 +1 @@
+module.exports = 'a';
```

Comment:
Minor: let's differentiate the content of these two files. E.g. have `a_main` export `'a main'` and `a-sub` export `'a sub'`.